shared/testutil/log.go

```diff
@@ -45,10 +45,10 @@ func assertLogs(t *testing.T, hook *test.Hook, want string, flag bool) {
 // WaitForLog waits for the desired string to appear the logs within a
 // time period. If it does not appear within the limit, the function
 // will throw an error.
-func WaitForLog(t *testing.T, hook *test.Hook, want string) {
+func WaitForLog(t *testing.T, hook *test.Hook, want string, durationCount int) {
```

Comment:
What is duration count? Can you please use a better variable name. 

```suggestion
func WaitForLog(t *testing.T, hook *test.Hook, want string, wait time.Duration) {
```